### PR TITLE
doc: Update the deployment guide using kestra

### PR DESCRIPTION
**Pull request by anna-geller**
Update the deployment page to show the dedicated CloudQuery plugin in kestra based on a simple lightweight example.

### CODE REVIEW

File: website/pages/docs/deployment/kestra.mdx [hunk 1]
```diff
@@ -1,102 +1,86 @@
...
+          path: cloudquery/file
+          version: v3.4.17
+          spec:
+            path: "cloudquery/{{TABLE}}/{{UUID}}.{{FORMAT}}"
+            format: csv
 ```
 
-We are using the Docker runner with a `Bash` task to run the `cloudquery sync` command. The `inputFiles` section allows us to pass a configuration file to the task. It is also possible to read this configuration file from disk or a remote location, but we will keep it simple for now.
-
-import { Callout } from 'nextra-theme-docs';
```
Comment (erezrokah) [inline website/pages/docs/deployment/kestra.mdx:70]: I believe you need to keep this import to use the `Callout` component. That should be the failing Vercel build

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks for the PR @anna-geller, added a comment regarding the failing Vercel build. I'll take another look on the deployed version once the build is passing

File: website/pages/docs/deployment/kestra.mdx (hunk 1 above)
Comment (anna-geller) [reply website/pages/docs/deployment/kestra.mdx:70]: thanks, done

File: website/pages/docs/deployment/kestra.mdx [hunk 2]
```diff
@@ -1,102 +1,88 @@
...
+description: Learn how to run CloudQuery as a Kestra flow, using the HackerNews API as a source and CSV file as a destination.
 tag: tutorial
-date: 2023/04/21
+date: 2024/01/22
 ---
 
 # Orchestrating CloudQuery Syncs with Kestra
 
-In this tutorial, we will show you how to run CloudQuery as a [Kestra](https://kestra.io/) flow, using the AWS source- and Postgresql destination plugins as an example. Kestra is an open source orchestration tool that allows you to schedule and monitor CloudQuery syncs. 
+This tutorial shows how to run CloudQuery as a [Kestra](https://kestra.io/) flow, using the the HackerNews API as a source and CSV file as a destination. Kestra is an open-source event-driven orchestrator, which has a [dedicated CloudQuery plugin](https://kestra.io/plugins/plugin-cloudquery).
```
Comment (erezrokah) [inline website/pages/docs/deployment/kestra.mdx:10]:
```suggestion
This tutorial shows how to run CloudQuery as a [Kestra](https://kestra.io/) flow, using the HackerNews API as a source and CSV file as a destination. Kestra is an open-source event-driven orchestrator, which has a [dedicated CloudQuery plugin](https://kestra.io/plugins/plugin-cloudquery).
```

**Review by erezrokah — APPROVED** [review]
Looks great, thanks for the update 🚀

**Pull request merged**